plugins/de.cognicrypt.staticanalyzer/src/de/cognicrypt/staticanalyzer/results/ResultsCCUIListener.java

```diff
@@ -230,6 +245,27 @@ public void reportError(final AbstractError error) {
 		}
 
 	}
+	
+	@SuppressWarnings("unchecked")
+	public Boolean isLegacyClient() {
+		final String path = this.currentProject.getLocation().toOSString() + Constants.innerFileSeparator + Constants.SECURE_COMMUNICATION + Constants.JSON_EXTENSION;
+		JSONParser parser = new JSONParser();
+		try {
+			if (Files.exists(Paths.get(path))) {
+
+					Object obj = parser.parse(new InputStreamReader(new FileInputStream(path)));
+					JSONObject jsonObject = (JSONObject) obj;
+					String clientsQuestion = "Which clients should the server at least support?";
```

Comment:
please use string constants here that get used for this code and the JSON creation.